examples/demo.js

```diff
@@ -6,6 +6,30 @@ var defaultHtmlSequencerUi = require('./lib/defaultHtmlSequencerUi.js'),
   insertPreview = require('./lib/insertPreview.js');
 
 window.onload = function () {
+  const toggleSwitch = document.querySelector('.theme-switch input[type="checkbox"]');
+  const currentTheme = localStorage.getItem('theme');
+
+  if (currentTheme) {
+    document.documentElement.setAttribute('data-theme', currentTheme);
+  
+  if (currentTheme === 'dark') {
+    toggleSwitch.checked = true;
+  }
+  }
+
+  function switchTheme(e) {
```

Comment:
Can this be converted to a named function variable?
```js
const switchTheme = function(e) {}
```
or arrow function
```js
const switchTheme = e => {
}
```

This will prevent hoisting and increase efficiency. More about it [here](https://medium.com/javascript-in-plain-english/https-medium-com-javascript-in-plain-english-what-is-hoisting-in-javascript-a63c1b2267a1)